### DISCUSSION TITLE
osu! Lazer Does not “capture” media controls

### DISCUSSION THREAD

**Discussion by AndrewHack242** [ppy/osu#19060] [Ideas] [answered]
While osu! lazer does respond to the media controls on my keyboard (play, pause, next song, previous song), it does not capture the input. As a result, I often end up accidentally also skipping songs on Spotify and causing spotify to play if I'm using the media controls on the main menu.

If I'm tabbed into osu! It probably shouldn't allow other apps to receive the media inputs.

I am on Windows 10.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/5918
